src/e2e-app/BUILD.bazel

```diff
@@ -90,17 +90,14 @@ sass_binary(
 ts_devserver(
     name = "devserver",
     testonly = True,
-    # Root paths can be used simplify the loading of files from external Bazel repositories
-    # (such as the Bazel managed deps repository called "npm")
     additional_root_paths = [
         "npm/node_modules",
     ],
     # Name of the AMD module that should be required on page load.
     entry_module = "angular_material/src/e2e-app/main",
     port = 4200,
-    # Scripts which will be included in the serving_path bundle after "require.js" has been
-    # loaded.
-    # TODO(jelbourn): remove UMDs from here once we don't have to manually include them
```

Comment:
I removed this TODO since it makes sense to exclude them manually (and we already made this less hard-coded). Also removed a few comments that were not necessary since it's obvious and we don't do it for other devserver targets.